src/i18n/ES/grid-resources.ts

```diff
@@ -98,7 +98,17 @@ const GridResourceStringsES_: ExpandRequire<IGridResourceStrings> = {
   igx_grid_hiding_uncheck_all_label: 'Show All',
   igx_grid_pinning_check_all_label: 'Pin All',
   igx_grid_pinning_uncheck_all_label: 'Unpin All',
-  igx_grid_toolbar_actions_filter_prompt: 'Filter columns list ...'
+  igx_grid_toolbar_actions_filter_prompt: 'Filter columns list ...',
+  igx_grid_snackbar_addrow_label: 'Row added',
+  igx_grid_snackbar_addrow_actiontext: 'SHOW',
+  igx_grid_actions_edit_label: 'Edit',
+  igx_grid_actions_add_label: 'Add',
+  igx_grid_actions_add_child_label: 'Add Child',
```

Comment:
This **igx_grid_actions_add_child_label** string exists in the _igniteui-angular-i18n_ repository and yet do not in the product repository _igniteui-angular_. I expect is either an excess one or awaits implementation?